__integrations__/__tests__/updateRCConfiguration.test.ts

```diff
@@ -0,0 +1,151 @@
+import * as TE from "fp-ts/lib/TaskEither";
+
+import { CosmosClient, CosmosClientOptions, Database } from "@azure/cosmos";
+import { pipe } from "fp-ts/lib/function";
+import { getNodeFetch } from "../utils/fetch";
+import {
+  REMOTE_CONTENT_COSMOSDB_KEY,
+  REMOTE_CONTENT_COSMOSDB_NAME,
+  REMOTE_CONTENT_COSMOSDB_URI
+} from "../env";
+import {
+  createRCCosmosDbAndCollections,
+  fillRCConfiguration
+} from "../__mocks__/fixtures";
+import {
+  aNewRemoteContentConfiguration,
+  aRemoteContentConfiguration
+} from "../__mocks__/mock.remote_content";
+
+const baseUrl = "http://function:7071";
+
+export const aRemoteContentConfigurationList = [aRemoteContentConfiguration];
+
+const cosmosClient = new CosmosClient({
+  endpoint: REMOTE_CONTENT_COSMOSDB_URI,
+  key: REMOTE_CONTENT_COSMOSDB_KEY
+} as CosmosClientOptions);
+
+// eslint-disable-next-line functional/no-let
+let database: Database;
+
+beforeAll(async () => {
+  database = await pipe(
+    createRCCosmosDbAndCollections(cosmosClient, REMOTE_CONTENT_COSMOSDB_NAME),
+    TE.getOrElse(() => {
+      throw Error("Cannot create db");
+    })
+  )();
+  await fillRCConfiguration(database, aRemoteContentConfigurationList);
+});
+
+describe("UpdateRCConfiguration", () => {
+  test("should return a 400 error if the payload is not valid", async () => {
+    const aFetch = getNodeFetch({});
+    const body = {};
+    const r = await putCreateRCConfiguration(aFetch)(
+      body,
+      aRemoteContentConfiguration.configurationId,
+      aRemoteContentConfiguration.userId
+    );
+    const jsonResponse = await r.json();
+
+    expect(r.status).toBe(400);
+    expect(jsonResponse.title).toBe("Invalid NewRCConfigurationPublic");
+  });
+
+  test("should return a 400 error if the payload is valid but the configurationId is not", async () => {
+    const aFetch = getNodeFetch({});
+    const body = aNewRemoteContentConfiguration;
+    const r = await putCreateRCConfiguration(aFetch)(
+      body,
+      "invalidUlid",
+      aRemoteContentConfiguration.userId
+    );
+    const jsonResponse = await r.json();
+
+    expect(r.status).toBe(400);
+    expect(jsonResponse.title).toBe(
+      'Invalid string that matches the pattern "^[0-9a-hjkmnp-tv-zA-HJKMNP-TV-Z]{26}$"'
+    );
+  });
+
+  test("should return a 403 error if the header x-user-id is not defined", async () => {
+    const aFetch = getNodeFetch({});
+    const body = aNewRemoteContentConfiguration;
+    const r = await putCreateRCConfiguration(aFetch)(
+      body,
+      aRemoteContentConfiguration.configurationId
+    );
+    const jsonResponse = await r.json();
+
+    expect(r.status).toBe(403);
+    expect(jsonResponse.title).toBe("Anonymous user");
```

Comment:
Can we change this response title? `Anonymous user` is too generic. Maybe something like `Access denied to anonymous user`.